### DISCUSSION TITLE
After update to 2.0.21 dark mode not working

### DISCUSSION THREAD

**Discussion by VOOM108** [kimai/kimai#4034] [Questions & Answers]
I updated from an early 2.0.x to 2.0.21 and now the dark theme does not work anymore. I set it in my user settings, but nothing changes. Any idea, why that may happen and what I can do? 

Other than that, the update went smoothly. The only unusual thing was that when refreshing the cache, the server would not do it, complaining about not enough memory. So I removed the cache as advised in the docs. Could that be the cause?

**Comment by kevinpapst** [comment]
Sounds as if the failed "cache warm" prevented the assets from being copied.

Execute `bin/console assets:install`

**Comment by VOOM108** [comment]
I remember, I had to do that before once as well. In this case, the command was executed without errors, but the dark mode still is not there...

**Comment by kevinpapst** [comment]
Closing, fixed by browser switch, see #4033
